### DISCUSSION TITLE
container/storage

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#210] [General]
# container/storage

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/container/storage.html

**Comment by shihaitao2005** [comment]
Docker 支持 3 中挂载方式：bind mount、volume、tmpfs mount。应该更改为 3种

**Reply by isno** [reply]
谢谢，已修正。

**Comment by LFlops** [comment]
"目标是位容器编排系统" -> "目标是为容器编排系统"

**Reply by isno** [reply]
谢谢指正，已修改。

**Comment by zjyyzhengli** [comment]
1. “7.5.6 Kubernete**~~r~~**s 的存储系统设计” -- 多了一个字母“r”
2. “如果 Pod 中使用的是 EmptyDir、HostPath 这类普通 Volume，并不会经历**附着**/分离的操作” -- 这里的“附着”建议和上文的翻译（上文是翻译为“附加”）统一，这样便于理解。
3. “可以看出，接口中定义的操作就是**图 7-27** Master 节点中 准备（Provision）和 附加（Attach）的逻辑。” -- 这里应该指的是**图7-24**

**Reply by zjyyzhengli** [reply]
“普通的 Volume：用于临时数据存储，生命周期与 Pod 绑定，如 emptyDir、hostPath 等。**这类 Volume 在 Pod 重启时数据会丢失**，适用于无需持久化的数据场景。” -- 这里建议改成 “这类 Volume 在 Pod 重启**或Pod 被调度到另外一台节点时**，数据会丢失” 会更加准确。
